### DISCUSSION TITLE
percona-xtrabackup in homebrew-core failing bump automations

### DISCUSSION THREAD

**Discussion by jkrauska** [Homebrew/discussions#4530] [Everyday usage]
### Output of `brew config`

```shell
n/a CI/CD Issue
```

### Output of `brew doctor`

```shell
n/a CI/CD Issue
```

### Description of issue

It seems recent version bump PRs for percona-xtrabackup have been failing.

eg.
https://github.com/Homebrew/homebrew-core/pull/131036

Any insight in to what's blocking these PRs?

**Comment by SMillerDev** [comment]
CI says it failed to build since patches didn't apply anymore. If you want a newer version please open a pull request that resolves the issues.

**Reply by jkrauska** [reply]
Thanks.. Making an attempt. 
https://github.com/Homebrew/homebrew-core/pull/132034

It looks to me like older macos versions might no longer be supported, but I'll try to better understand and document this.
